react/training/src/app/screens/Login/components/layout.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
```

Comment:
This is another component. This should be in a folder like `LoginForm/index.js`. So the result will be `Login/components/LoginForm/index.js`